src/main/java/ai/rev/speechtotext/helpers/ClientHelper.java

```diff
@@ -0,0 +1,29 @@
+package ai.rev.speechtotext.helpers;
+
+import ai.rev.speechtotext.ApiInterceptor;
+import ai.rev.speechtotext.ErrorInterceptor;
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.converter.scalars.ScalarsConverterFactory;
+
+public class ClientHelper {
```

Comment:
Can this be used by the async api client as well?

Reply:
Yes! That was the intention. Good catch.